environment.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   # For running
   - gdal
   - geopandas
+  - pandas>=1.3.5,<1.4
```

Comment:
I think this could be simplified with `pandas=1.3`, but they are effectively the same and this is more explicit, so either is fine by me.

Reply:
(and would need to be `pandas==1.3` in `setup.py`)